logprep/util/pseudo/encrypter.py

```diff
@@ -82,13 +83,13 @@ def encrypt(self, input_str: str) -> str:
         session_key_enc_enc: bytes = cipher_rsa_depseudo.encrypt(session_key_enc)
 
         # concatenate, encode, and return
-        return base64.b64encode(
-            session_key_enc_enc
-            + aes_key_depseudo.nonce
-            + depseudo_key_enc
-            + aes_key_input_str.nonce
-            + input_str_enc
-        ).decode("ascii")
+        return (
+            f"{base64.b64encode(session_key_enc_enc).decode('ascii')}:"
+            f"{base64.b64encode(aes_key_depseudo.nonce).decode('ascii')}:"
+            f"{base64.b64encode(depseudo_key_enc).decode('ascii')}:"
+            f"{base64.b64encode(aes_key_input_str.nonce).decode('ascii')}:"
+            f"{base64.b64encode(input_str_enc).decode('ascii')}"
+        )
```

Comment:
how about you implement this as `__str__` or `__repr__` method inside the `GCMPseudonym` dataclass?